beta-2-setup.md

```diff
@@ -181,14 +179,13 @@ The configuration files for the Ansible installer are currently available on
 Github. Clone the repository:
 
     cd
-    git clone https://github.com/detiber/openshift-ansible.git
+    git clone https://github.com/detiber/openshift-ansible.git -b v3-beta2
     cd ~/openshift-ansible
-    git checkout v3-beta2
 
 ### Configure Ansible
 Move the staged Ansible configuration files to `/etc/ansible`:
 
-    mv -f ~/training/beta2/ansible/* /etc/ansible
+    "cp" -r ~/training/beta2/ansible/* /etc/ansible/
```

Comment:
Probably causes no issues here but download-and-mv is something to avoid because files will end up with the wrong SELinux context, while cp applies the expected SELinux label while creating the destination files.